static/app/components/events/aiAutofix/index.spec.tsx

```diff
@@ -43,4 +48,32 @@ describe('AiAutofix', () => {
     expect(screen.getByText('Fixed the bug!')).toBeInTheDocument();
     expect(screen.getByText('View Pull Request')).toBeInTheDocument();
   });
+
+  it('renders the correct steps', () => {
+    render(
+      <AiAutofix
+        group={{
+          ...group,
+          metadata: {
+            autofix: {
+              status: 'COMPLETED',
+              completed_at: '',
+              created_at: '',
+              steps: [
+                {
+                  id: '1',
+                  title: 'I am processing',
+                  description: 'oh yes I am',
+                  status: 'PROCESSING',
+                },
+              ],
+            },
+          } as EventMetadataWithAutofix,
+        }}
+      />
+    );
+
+    expect(screen.getByText('I am processing')).toBeInTheDocument();
+    expect(screen.getByText('oh yes I am')).toBeInTheDocument();
```

Comment:
Would love to see this be more accessible.

I think you could use [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) to have the heading be described by the text below (or maybe even [`aria-details`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details))

cc @vuluongj20 wdyt?